spec/services/organization_update_service_spec.rb

```diff
@@ -14,7 +14,7 @@
     end
 
     context "when object is invalid" do
-      it "should not update and return false" do
+      it "should not update, return false" do
```

Comment:
The previous sentence made more sense. :smile: Does this mean "it should not update, and it should return false", or "it should not update and it should NOT return false"?